playbooks/roles/logstash/defaults/main.yml

```diff
@@ -3,6 +3,9 @@
 logstash_elasticsearch_hosts:
   - http://localhost:9200
 
+logstash_elasticsearch_user: logstash_system
```

Comment:
Can we document the process for the manual steps required for setting this up?
We can link to the elasticsearch documentation for the steps required so we don't have to describe them in excruciating detail here, but in the README's for the roles we should outline what needs to be done manually when setting up a new ELK server.

Reply:
@kaizoku, I've added some notes on README files of ``kibana``, ``elasticsearch`` and ``logstash`` role. Can you check if those seem sufficient?